src/handler/fadingMessageHandler.js

```diff
@@ -20,9 +20,9 @@ const fadingMessageDeleteLoop = async function(client) {
 
         try {
             let guild = await client.guilds.fetch(fadingMessage.guildId);
-            let channel = guild.channels.cache.get(fadingMessage.channelId);
+            let channel = await guild.channels.cache.get(fadingMessage.channelId);
             /** @type {Message} */
-            let message = channel.messages.cache.get(fadingMessage.messageId);
+            let message = await channel.messages.fetch(fadingMessage.messageId);
```

Comment:
aha also doch fetch :eyes:

Reply:
das is aber nicht das was du gemeint hast >:(